koku/api/report/queries.py

```diff
@@ -1021,9 +1021,9 @@ def _group_by_ranks(self, query, data):  # noqa: C901
             if rank_value not in rankings:
                 rankings.append(rank_value)
                 distinct_ranks.append(rank)
-        return self._ranked_list(data, distinct_ranks)
+        return self._ranked_list(data, distinct_ranks, list(rank_annotations.keys()))
```

Comment:
Passing a dictionary into `list()` will return a list of keys.
```suggestion
        return self._ranked_list(data, distinct_ranks, list(rank_annotations))
```